### DISCUSSION TITLE
Compare with other frameworks/libs

### DISCUSSION THREAD

**Discussion by Igloczek** [aidenybai/lucia#241] [General]
Hi, I'm looking for some comparisons to other frameworks/libs available on the market, to get what are the benefits or drawback coming with choosing Lucia over mentioned in the acknowledgments section Alpine, or recently released [petite-vue](https://github.com/vuejs/petite-vue), since all three seems to be very similar.

**Comment by aidenybai** [comment]
They're all pretty much the same, in terms of "lightweight-ness," Lucia is a good choice. In terms of developer ecosystem/popularity, Alpine and petite-vue will do the job.
